### ISSUE TITLE
Pi setup script zsh errors

### ISSUE THREAD

**Issue by Racerx323** [subtlepseudonym/pi#1]
Hi, Thanks for the great article about K8S on Raspberry Pi. I used your Pi setup script and received the following error after reboot. Is this of concern?
Error: /home/ama/.dotfiles/nvm/config.zsh:source:4: no such file or directory: /usr/share/nvm/init-nvm.sh

**Comment by subtlepseudonym** [comment]
I'm glad you liked it!

You're seeing that error because of some updates I made to my dotfiles, unrelated to pi development. You can prevent that error by removing `/home/ama/.dotfiles/nvm/config.zsh` if you're not using nvm. If you are, update the last line of that file to point to where you've installed your `init-nvm.sh` script.

More generally, if you're setting up additional pi's, you can prevent installation of dotfiles modules you're not going to use by adding `--exclude {module_i_dont_care_about}` to [this line](https://github.com/subtlepseudonym/pi/blob/master/setup/pi_raspbian.sh#L57) in the installation script. The module names come from the folders [in this repo](https://github.com/subtlepseudonym/dotfiles). For example, you can update the line to something like this:
```
/usr/bin/sudo -H -u ${user} ${home}/dotfiles/setup/install --exclude go --exclude macos --exclude nvm
```

**Comment by subtlepseudonym** [comment]
This issue should be resolved as of [aa8e1a5](https://github.com/subtlepseudonym/pi/commit/aa8e1a5e0c4d7cbd3164020d43a15c8de83c8b6f)

**Comment by Racerx323** [comment]
Thank you!
